tc/benchmarks/README.md

```diff
@@ -0,0 +1,40 @@
+# Building
+
+These benchmarks are automatically built when ```WITH_CAFFE2=ON``` is passed.
+If you have been following the instructions given [here](https://facebookresearch.github.io/TensorComprehensions/installation.html), you can use the command:
+
+```
+BUILD_TYPE=Release WITH_CAFFE2=ON CLANG_PREFIX=$(${CONDA_PREFIX}/bin/llvm-config --prefix) ./build.sh
+```
+
+# Running the autotuner manually
+By default a full evolutionary search is run with 25 generations and 100 candidates per generation. This will take some time for some of the kernels. This setting be changed by using the proper gflags options: ```--tuner_gen_generations``` and ```--tuner_gen_pop_size```.
+
+For instance, a more reasonable search could iterate as such:
```

Comment:
This is not a "more reasonable search", but a "shorter tuning session".  Let's not send a message that our default options are unreasonable